src/Sarif.Converters.UnitTests/AndroidStudioConverterTests.cs

```diff
@@ -213,6 +213,15 @@ public void AndroidStudioConverter_ConvertSarifResult_DoesNotRecordTopLevelEntry
             loc.PhysicalLocation.Should().BeNull();
         }
 
+        private static void ValidateLogicalLocations(IList<LogicalLocation> expectedLogicalLocations, IList<LogicalLocation> actualLogicalLocations)
```

Comment:
>ValidateLogicalLocations [](start = 28, length = 24)

If you derive `AndroidStudioConverterTests` and `FxCopConverterTests` from a new `ToolFileConverterTestsBase` class (not to be confused with `ToolFileConverterBaseTests`!), you can DRY this out by moving it to the base class.

Reply:
Yes. But I generally do not introduce a new layer in the class hierarchy to share a single helper. YMMV. I've converted this code to at least utilize a shared static helper.

---
In reply to: [243865843](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243865843) [](ancestors = 243865843)